services/payments/tests/unit/test_db_payments_users_repo.py

```diff
@@ -49,21 +49,6 @@ def app_environment(
     )
 
 
-async def _insert_and_get_row(
-    conn, table: sa.Table, values: dict[str, Any], pk_col: sa.Column, pk_value: Any
-):
-    result = await conn.execute(table.insert().values(**values).returning(pk_col))
-    row = result.first()
-    assert row[pk_col] == pk_value
-
-    result = await conn.execute(sa.select(table).where(pk_col == pk_value))
-    return result.first()
-
-
-async def _delete_row(conn, table, pk_col: sa.Column, pk_value: Any):
-    await conn.execute(table.delete().where(pk_col == pk_value))
-
-
 @pytest.fixture
 async def user(
```

Comment:
Would it not make sense to move these fixtures to `pytest-simcore` too? I think these would be useful when testing e.g. the api-server

Reply:
@bisgaard-itis they are already there. The idea here is overriding the `users` fixture from the `faker_users_data` plugin. I should not create yet another fixture with the same name in a different plugin since it would make it more confusing. Instead I provide you with a helper `insert_and_get_row_lifespan` to do it yourself in your services/package tests